CHANGELOG.rst

```diff
@@ -1,8 +1,8 @@
 Changelog
 =========
 
-1.17.0 (TBD)
-------------
+1.17.0 (2025-01-10)
```

Comment:
I'm missing b8bfb7d94849ec2fa61ffb3dadb6275089b07e0a here but I don't think we should include it in the changelog.